### ISSUE TITLE
Improve the way thumbs up/down system works on modding discussions

### ISSUE THREAD

**Issue by Rolniczy** [ppy/osu-web#4628]
Disclaimer: I'm new to this site yes. Before writing this I tried to take a look on multiple pulls centered around thumbs up/down topic but everything seems inactive/unresolved thus I write it with potential solution to once and for all fix this.

So basically everyone can upvote modding posts using thumb up no matter if post was helpful or no. This often leads to kudosu abuse cause modder can earn more kudosu than he should (you can like gather few people to purposely upvote everything to make this happen) which causes rather serious kudosu inflation. And because of that this can lead to silencing the modder (like in well known Kroytz's case). My idea is to rework the way thumbs up work - everyone can upvote it like it used to be, **but** only thumb up from mapset host (and/or any osu!team member) will indicate a kudosu grant, like it used to be in modding v1 I believe (I'm too young at mapping scene so i dont fully know how v1 worked but yea). This simple fix will massively reduce kudosu inflation/abuse cases without any negative outcome (but you may ask how people who made guest diffs will now grant kudosu to modder, well mapset host needs to close those posts anyways and he can judge easily if mod was accepted or no in meantime so he can grant it on his own)

I also wanna bring here the thumb down topic. Idea is to let anyone use it to express their opinion whether post was helpful or no, it will provide a bit more accurate feedback on questionable mod posts like veto or such without people expressing their opinions with responses below it. For current while it's pointless feature because except BNs/NATs etc no one can use it, enlightening that button for everyone will improve gathering opinions based on statistics/numerals

**Comment by Cynplytholowazy** [comment]
In my opinion kudosus should only be awarded by the mappers themselves and the GDers who participated in the set (and probably BNs and NATs), like how the old modding system works. That can ensure other people won't bloat the kudosu count of a particular user.

The GDer thing would bring another problem tho: how to define who is a GDer and who is not; but that would require another thread to point that out so not going in depth here.

For the thumbs down thing tho I guess it's originally to prevent abuses of it like targeting a certain person or certain posts etc. I dont really suggest having it open to the public because it could impact modders' motivation to continue modding and contribute to the community.

To sum up I think kudosus should only be awarded from thumbs up by the mapper or people who contributed into the set, while the thumbs up count can keep, but I don't really advise opening the thumbs down count.

More: I really wonder why the thumbs down exist since BNs and NATs can simply deny kudosu if they think the post does not worth kudosus instead of downvoting the comment so that it lose kudosus. + If the post has like 10 thumbs up having 1 or 2 thumbs down doesn't really do much.

**Comment by Naxesss** [comment]
fwiw this is a duplicate of an issue brought up 2 years ago here https://github.com/ppy/osu-web/issues/1698 which was written a few days after modding v2 was released, which raises the same concerns and solutions (apart from the downvote thing).

https://github.com/ppy/osu-web/issues/2165 would probably need to happen before guest mappers can upvote, but this is in the backlog so I doubt we'll see any of this soon unless focus shifts.

See https://github.com/ppy/osu-web/issues/4502 which was raised after abuse and https://github.com/ppy/osu-web/issues/4502#issuecomment-496347664 for developer stance. Intention seems to be to keep regular users able to award kudosu and still tie kudosu to the upvote system, which I can't say I agree with at all, but goes in line with the previously expressed, and somewhat controversial, vision of having osu modding follow a stackoverflow-tier-approach, where after x kds you can perform y action (in this case upvoting, and presumably downvoting too).
